gremlin-core/src/main/java/org/apache/tinkerpop/gremlin/process/traversal/util/BytecodeHelper.java

```diff
@@ -177,7 +175,7 @@
 import java.util.stream.Stream;
 
 /**
- * Utility class for parsing {@link Bytecode}.
+ * Utility class for parsing {@link GremlinLang}.
  *
```

Comment:
Should this class be renamed to GremlinLangHelper?

Reply:
it should be removed later